content/blog/environment-variables-in-golang/index.md

```diff
@@ -14,28 +14,28 @@ This tutorial assumes you have:
 *   Latest Golang version installed on your system
 *   A few minutes of your time.
 
-In this article, we will see what are the usage of environment variables in an application, and how to access the environment variables in a Go application with different inbuilt and third-party packages.
+In this article, we will know about environment variables and why to use them. And will access them in a Go application using inbuilt and third-party packages.
 
 ## What are environment variables?
 
-Environment variables are key-value pair on a system-wide level, that can be accessed by running processes on the system. Environment variables are often used to make the same program behave differently in separate deploy environments like PROD, DEV, or TEST. 
-Storing configuration in the environment is one of the principles of a twelve-factor app it enables applications to be built with portability.
+Environment variables are key-value pair on a system-wide level, and running processes can access that. These are often used to make the same program behave differently in different deploy environments like PROD, DEV, or TEST. 
+Storing configuration in the environment is one of the principles of a twelve-factor app it enables applications to be portable.
 
 ## Why should you use environment variables
 
-- If you are entering the sensitive information in the code, then all the unauthorized users who have access the code will have the sensitive information, you might not want that.
+- If you are entering the sensitive information in the code, then all the unauthorized users who have access the code will have the sensitive data, you might not want that.
 
-- If you are using the code versioning tool like `git`, you may push your DB credentials with the code and it will go public.
+- If you are using the code versioning tool like `git`, you may push your DB credentials with the code, and it will go public.
 
-- If you are managing these variables in one place, In case of any changes, you don't have to change it in all the places in application code it has been used. 
+- If you are managing variables in one place. In case of any changes, you don't have to change it in all the places in application code. 
```

Comment:
shouldn't this be? `If you are managing variables in one place, in case of any changes, you don't have to change it in all the places in application code. `